ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/mainnet/precompiles/privacy/PrivacyPrecompiledContract.java

```diff
@@ -102,7 +101,7 @@ public BytesValue compute(final BytesValue input, final MessageFrame messageFram
     ReceiveResponse receiveResponse;
     try {
       receiveResponse = enclave.receive(receiveRequest);
-    } catch (IOException e) {
+    } catch (Exception e) {
       LOG.debug("Enclave probably does not have private transaction with key {}.", key, e);
```

Comment:
Existing code, but should this log at a higher level error? or is this a normal, common use case.  I would expect that if it's normal/expected, then logic would pick those out and log at debug. but for everything else, error level and maybe rethrow?

Reply:
no rethrow here - this is how the precompile should behave.